config/types.go

```diff
@@ -249,11 +253,18 @@ func (a *AgentServiceConnectProxyConfig) ToConsulType() *api.AgentServiceConnect
 	return result
 }
 
+func (a *AgentServiceConnectProxyConfig) GetPrimaryListenerPort() int {
```

Comment:
Just to be consisent with the naming.
```suggestion
func (a *AgentServiceConnectProxyConfig) GetPublicListenerPort() int {
```